.docker-compose/docker-compose.tpl.yml

```
@@ -0,0 +1,161 @@
+services:
+  traefik:
+    image: "traefik:v3.1"
+    container_name: "traefik"
+    restart: "on-failure"
+    command:
+      # - "--log.level=TRACE"
+      - "--accessLog=true"
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false" # services need to be exposed by setting traefik.enable=true
+      - "--entryPoints.websecure.address=:443"
+      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
+      #- "--certificatesresolvers.myresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory" # used for cert debugging
+      - "--certificatesresolvers.myresolver.acme.email=postmaster@cometdxp.com"
+      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
+    ports:
```

```suggestion
    ports:
```

might also need http for acme challenge